### PR TITLE
Fixing links to suse.com/documentation to correct URL (noref)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/deployment_cifs.xml [hunk 1]
```diff
@@ -198,7 +198,7 @@
    </para>
    <para>
     For details about &sle; &hasi;, see
-    <link xlink:href="https://www.suse.com/documentation/sle-ha-15/"/>.
+    <link xlink:href="https://https://documentation.suse.com/sle-ha/15-SP1/"/>.
```
Comment (tbazant) [inline xml/deployment_cifs.xml]:
```suggestion
    <link xlink:href="https://documentation.suse.com/sle-ha/15-SP1/"/>.
```

**Comment by lproven** [comment]
>  Set up a basic 2-node HA cluster with shared storage as described in

For numbers under 9 I suggest spelling out the word: "two-node"

**Comment by lproven** [comment]
>      <link xlink:href="https://documentation.suse.com/sle-ha/15-SP1/single-html/SLE-HA-install-quick/#art-sleha-install-quick"/>.

I think we do not need the anchor here (the part after the # symbol) since the link is to the whole book.

**Comment by asettle** [comment]
> > Set up a basic 2-node HA cluster with shared storage as described in
> 
> For numbers under 9 I suggest spelling out the word: "two-node"

Since this PR is specifically for links, I want to keep track of that *only* (not including the feedback section), so the backport is clean. Feel free to make this change separately :)

**Comment by lproven** [comment]
> > > Set up a basic 2-node HA cluster with shared storage as described in
> > 
> > 
> > For numbers under 9 I suggest spelling out the word: "two-node"
> 
> Since this PR is specifically for links, I want to keep track of that _only_ (not including the feedback section), so the backport is clean. Feel free to make this change separately :)

OK, good point. Fair enough then. Please have this **LGTM** and use it wisely.

**Pull request merged**